mscore/file.cpp

```diff
@@ -2540,76 +2540,80 @@ bool MuseScore::saveSvg(Score* score, const QString& saveName)
             p.translate(-r.topLeft());
 
       foreach (Page* page, score->pages()) {
```

Comment:
OK.  I stole that code from elsewhere in MuseScore many moons ago.  So for (a : b) is more efficient?  I might adjust a couple of comments too, while I'm in there...